client/my-sites/checkout/checkout/duplicate-product-notice-content.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+import { isArray } from 'lodash';
+import React, { FunctionComponent } from 'react';
+import { useSelector } from 'react-redux';
+
+/**
+ * Internal Dependencies
+ */
+import { getSitePurchases } from 'state/purchases/selectors';
+
+// TODO: update types
+type Props = {
+	product: any;
+	selectedSite: any;
+};
+
+const DuplicateProductNoticeContent: FunctionComponent< Props > = ( { product, selectedSite } ) => {
+	const translate = useTranslate();
+	const purchases = useSelector( ( state ) => getSitePurchases( state, selectedSite?.ID ) );
+	const purchase = isArray( purchases )
+		? purchases.find( ( p ) => p.productSlug === product.productSlug )
+		: null;
+	const purchaseId = purchase?.id;
+	const subscriptionUrl = purchaseId
+		? `/me/purchases/${ selectedSite.slug }/${ purchaseId }`
+		: '/me/purchases/';
+
+	return (
+		<div className="checkout__duplicate-notice">
+			<p className="checkout__duplicate-notice-message">
+				{ translate(
+					'You currently own %(product)s. The plan you are about to purchase also includes this product. Consider removing your {{link}}%(product)s subscription{{/link}}.',
+					{
+						args: {
+							product: product.productName,
+						},
+						components: {
+							link: <a href={ subscriptionUrl } />,
+						},
+					}
+				) }
+			</p>
+			<a className="checkout__duplicate-notice-link" href={ subscriptionUrl }>
+				{ translate( 'Manage Subscription' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **33** times:
`translate( 'Manage subscription' )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/plan-backup-warning-in-checkout)